src/H5webApp.tsx

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
-import { App, H5GroveProvider } from '@h5web/app';
+import React, { useEffect, useState } from 'react';
+import { App, getFeedbackMailto, H5GroveProvider } from '@h5web/app';
 import { ServerConnection } from '@jupyterlab/services';
 
+const FEEDBACK_EMAIL = 'h5web@esrf.fr';
+
+function TwoRenderApp() {
+  const [isFirstRender, setIsFirstRender] = useState(true);
+
+  useEffect(() => {
+    setIsFirstRender(false);
+  }, []);
+
+  if (isFirstRender) {
+    return null;
+  }
+
+  return (
+    <App
+      getFeedbackURL={(context) => getFeedbackMailto(context, FEEDBACK_EMAIL)}
+    />
+  );
+}
```

Comment:
For some reason, I had to reimplement this fix for 1.4.0. Otherwise, the two flex from reflex containers are set to `0/0` instead of `25/75`...